mlflow/pyfunc/__init__.py

```diff
@@ -684,7 +684,15 @@ def _validate_prediction_input(
                     f"Error: {e}",
                 )
 
+        # temp workaround for rag model in model serving passing convert_chat_responses
+        # to predict method, we shouldn't validate its schema
+        if self.loader_module == "mlflow.langchain":
+            convert_chat_responses = params.pop("convert_chat_responses", None) if params else None
+        else:
+            convert_chat_responses = None
         params = _validate_params(params, self.metadata)
+        if convert_chat_responses is not None:
+            params["convert_chat_responses"] = convert_chat_responses
```

Comment:
Instead, I think we should start allowing parameters that aren't in the signature to be passed. Can we make that change instead? I'm concerned about adding special case logic in pyfunc predict for specific flavors. If this misses the 2.13.0 release, that's okay.

Reply:
Ok, I think that makes sense for allowing more params skipping params validation, it's just currently convert_chat_responses must be set to true so it returns a dictionary as expected. We can revisit this PR later for general mlflow pyfunc tracing support.

Reply:
Sounds good!

Reply:
> Instead, I think we should start allowing parameters that aren't in the signature to be passed.

Then we can also add the callback as the parameters ?

Reply:
what about allowing `_validate_prediction_input` to be a overridable method in `model_impl` ? instead of adding a patch here.